### PR TITLE
Bump slf4j-api to 2.0.13, and Logback to 1.3.14

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -50,17 +50,25 @@ dependencies {
     // The classes of "embulk-spi" are excluded from "embulk-core" below for the consistency as a JAR file.
     // But, they should be included in the compile classpath.
     compileOnly "org.embulk:embulk-spi:0.11"
+    compileOnly "org.slf4j:slf4j-api:2.0.13"
+    compileOnly "org.msgpack:msgpack-core:0.8.24"
```
Comment (hiroyuki-sato) [inline build.gradle:54]: Just for interest. Why need the `msgpack-core` in this PR?

**Review by hiroyuki-sato — APPROVED** [review]
LGTM👍

File: build.gradle (hunk 1 above)
Comment (dmikurube) [reply build.gradle:54]: It has been included in `embulk-spi` implicitly. There is no substantial changes. Not needed literally.

Wanted to include this just to be explicit.

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**